server/controllers/matchChat.ts

```diff
@@ -0,0 +1,85 @@
+import { Request, Response } from 'express';
+import {
+  addMessageService,
+  deleteMessageService,
+  getAllMessagesService,
+  getMessageByIdService,
+  editMessageService,
+} from '../services';
+
+const addMessage = async (req: Request, res: Response) => {
```

Comment:
this controller missing types and returning types :
addMessage: RequestHandler = (): Promises<Void> =>